snuba/clickhouse/columns.py

```diff
@@ -1,475 +1,70 @@
-        return UInt(self.size)
-
-
-class Float(ColumnType[TModifiers]):
-    def __init__(self, size: int, modifiers: Optional[TModifiers] = None,) -> None:
-        super().__init__(modifiers)
-        assert size in (32, 64)
-        self.size = size
-
-    def _repr_content(self) -> str:
-        return str(self.size)
-
-    def __eq__(self, other: object) -> bool:
-        return (
-            self.__class__ == other.__class__
-            and self.get_modifiers() == cast(Float[TModifiers], other).get_modifiers()
-            and self.size == cast(Float[TModifiers], other).size
-        )
-
-    def _for_schema_impl(self) -> str:
-        return "Float{}".format(self.size)
-
-    def set_modifiers(self, modifiers: Optional[TModifiers]) -> Float[TModifiers]:
-        return Float(size=self.size, modifiers=modifiers)
-
-    def get_raw(self) -> Float[TModifiers]:
-        return Float(self.size)
-
-
-class Date(ColumnType[TModifiers]):
-    pass
-
-
-class DateTime(ColumnType[TModifiers]):
-    pass
-
-
-class Enum(ColumnType[TModifiers]):
-    def __init__(
-        self, values: Sequence[Tuple[str, int]], modifiers: Optional[TModifiers] = None,
-    ) -> None:
-        super().__init__(modifiers)
-        self.values = values
-
-    def _repr_content(self) -> str:
-        return ", ".join("'{}' = {}".format(v[0], v[1]) for v in self.values)
-
-    def __eq__(self, other: object) -> bool:
-        return (
-            self.__class__ == other.__class__
-            and self.get_modifiers() == cast(Enum[TModifiers], other).get_modifiers()
-            and self.values == cast(Enum[TModifiers], other).values
-        )
-
-    def _for_schema_impl(self) -> str:
-        return "Enum({})".format(
-            ", ".join("'{}' = {}".format(v[0], v[1]) for v in self.values)
-        )
-
-    def set_modifiers(self, modifiers: Optional[TModifiers]) -> Enum[TModifiers]:
-        return Enum(values=self.values, modifiers=modifiers)
+from snuba.utils.schemas import UUID, AggregateFunction, Any, Array, Column
+from snuba.utils.schemas import ColumnSet as BaseColumnSet
+from snuba.utils.schemas import (
+    ColumnType,
+    Date,
+    DateTime,
+    Enum,
+    FixedString,
+    FlattenedColumn,
+    Float,
+    IPv4,
+    IPv6,
+    Nested,
+    Nullable,
+    ReadOnly,
+    SchemaModifiers,
+    String,
+    TModifiers,
+    TypeModifier,
+    TypeModifiers,
+    UInt,
+)
 
-    def get_raw(self) -> Enum[TModifiers]:
-        return Enum(self.values)
+__all__ = (
+    "Any",
+    "AggregateFunction",
+    "Array",
+    "Column",
+    "ColumnSet",
+    "ColumnType",
+    "Date",
+    "DateTime",
+    "Enum",
+    "FixedString",
+    "FlattenedColumn",
+    "Float",
+    "IPv4",
+    "IPv6",
+    "Nested",
+    "Nullable",
+    "QualifiedColumnSet",
+    "ReadOnly",
+    "SchemaModifiers",
+    "String",
+    "TModifiers",
+    "TypeModifier",
+    "TypeModifiers",
+    "UInt",
+    "UUID",
+)
 
 
-class ColumnSet:
+class ColumnSet(BaseColumnSet):
```

Comment:
This is the only part that changed, we are inheriting from a BaseColumnSet that does nothing.